test/processors/base_processor_test.rb

```diff
@@ -184,9 +184,19 @@ def processor.ftp_upload_file(u,f,o); 'uploaded!'; end
     processor.original_format.must_equal 'ogg'
   end
 
-  it 'extracts format from uri' do
+  it 'extracts format from the uri' do
     uri = URI.parse 'file://what/test.mp3?wat=dat'
-    processor.extract_format(uri).must_equal 'mp3'
+    processor.format_from_uri(uri).must_equal 'mp3'
+  end
+
+  it 'extracts format from the file' do
+    processor.format_from_file(File.open(in_file('test.ogg'))).must_equal 'ogg'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.